frontend/src/views/web/landing.html

```diff
@@ -47,7 +47,7 @@ <h4>
                             <p>
                                 Explore other past, ongoing and upcoming challenges.
                             </p>
-                            <a class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-light fs-14 mt-12" ui-sref="web.challenge-main">View All</a>
+                            <a class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-light fs-14 mt-12" ui-sref="all-challenges">View All</a>
```

Comment:
One more additon, if user is logged in then redirect to 'web.challenge-main' othersiw 'all-challeges'. @xamfy

Reply:
@RishabhJain2018 can you please do this?

Reply:
I'm taking this @RishabhJain2018